ssh_covert.py

```
@@ -0,0 +1,146 @@
+        
+        if not connection["host"]:
+            print(f"警告: Host {host['name']} 缺少 HostName，已跳过")
+            continue
+            
+        ssh_connections.append(connection)
+    
+    return {"ssh_connections": ssh_connections}
+
+def main():
+    """主函数"""
+    # 使用测试配置文件（确保文件存在）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)